Makefile

```diff
@@ -1,107 +1,121 @@
-NAME=Libertinus
-VERSION=6.11
-
-DIST=$(NAME)-$(VERSION)
-
-SOURCEDIR=sources
-BUILDDIR=build
-GSUB=$(SOURCEDIR)/features/gsub.fea
-DOC=documentation
-TOOLS=tools
-
-PY?=python3
-BUILD=$(TOOLS)/build.py
-NORMALIZE=$(TOOLS)/sfdnormalize.py
-
-NULL=
-
-FONTS=Sans-Regular \
-       Sans-Bold \
-       Sans-Italic \
-       Serif-Regular \
-       Serif-Semibold \
-       Serif-Bold \
-       Serif-Italic \
-       Serif-SemiboldItalic \
-       Serif-BoldItalic \
-       SerifDisplay-Regular \
-       Math-Regular \
-       SerifInitials-Regular \
-       Mono-Regular \
-       Keyboard-Regular \
-       $(NULL)
-
-SFD=$(FONTS:%=$(SOURCEDIR)/$(NAME)%.sfd)
-NRM=$(FONTS:%=$(BUILDDIR)/$(NAME)%.nrm)
-CHK=$(FONTS:%=$(BUILDDIR)/$(NAME)%.chk)
-OTF=$(FONTS:%=$(NAME)%.otf)
-SVG=preview.svg
-PDF=$(DOC)/Opentype-Features.pdf $(DOC)/Sample.pdf $(DOC)/Math-Sample.pdf
+NAME = Libertinus
+VERSION = 6.11
+
+MAKEFLAGS := -s -j$(shell nproc) -Otarget
+SHELL = bash
+
+DIST = $(NAME)-$(VERSION)
+
+SOURCEDIR = sources
+BUILDDIR = build
+GSUB = $(SOURCEDIR)/features/gsub.fea
+DOCSDIR = documentation
+TOOLSDIR = tools
+
+PY ?= python3
+BUILD = $(TOOLSDIR)/build.py
+NORMALIZE = $(TOOLSDIR)/sfdnormalize.py
+
+# Default to explicitly enumerating fonts to build;
+# use `make ALLFONTS=true ...` to build all *.sfd files in the source tree or
+# use `make FONTS="Face-Style" ...` to make targets for only particular font(s).
+ALLFONTS ?= false
```

Comment:
Why do we need this at all? It is not likely that we will be adding any new styles any time soon, and even we did not updating the list here will be obvious mistake as the fonts wont be built.

Reply:
For your workflow, you probably don't need this. It serves two scenarios. First people like me that are messing around with the font can easily copy one of the source files to a new name and muck around in it while comparing it to the original, and we can have an option to build everything from sources without having to explicitly list what those are. Second it makes for a bit more flexible overall setup if copy/pasting and adapting for a new font project that might not be as set in stone yet as this one is. Your preferred workflow of building exactly the fonts specified for this project by default is retained.